pkg/detectors/browserstack/browserstack.go

```diff
@@ -22,11 +24,9 @@ var _ detectors.Detector = (*Scanner)(nil)
 const browserStackAPIURL = "https://www.browserstack.com/automate/plan.json"
 
 var (
-	defaultClient = common.SaneHttpClient()
```

Comment:
As a future maintainer, I'd find it helpful to see a comment (somewhere) explaining that `SaneHttpClient` doesn't appear to work for unknown reasons (even with the cookie jar attached).